### ISSUE TITLE
Update tooltip value

### ISSUE THREAD

**Issue by pvictor** [vasturiano/cartogram-chart#2]
Hello,
Thanks for this amazing library, super easy to use !
I've made a wrapper of your library to use with the R language (https://www.r-project.org/), project is here : https://github.com/pvictor/topogRam.

My issue is when you update the value used to distort topology, tooltip isn't updated, an example here : https://pvictor.github.io/NZ-topogRam/

Basically what i'm doing is : 

```
carto = Cartogram()
          .width(width)
          .height(height)
          .topoJson(shape)
          .topoObjectName('states')
          .projection(projection)
          .iterations(20)
          .value(function(d) {
           return d.properties.value;
          })
          .color(function(d) {
            return colorScale(d.properties.value);
          })
          .label(function(d) {
            return "Population of" + d.properties.name + "(" + d.properties.value + ")";
          })
          .valFormatter(d3.format(".3s"))
          //.onClick(function(d) {console.info(d)})
          (document.getElementById(el.id));

var selectValue = 'value';
$('#select').on('change', function() {
 selectValue = this.value;
 carto
   .value(function(d) {
       return d.properties[selectValue];
   });
 });
```

What i am missing ?

If you want a more thorough example let me know.

Thanks for your help !

Victor

**Comment by pvictor** [comment]
Thanks for your reply. This doesn't seems to work or I'm doing someting wrong (my JS skills are limited)

Here an adaptation of your world example : 

```html
<head>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/4.11.0/d3.min.js"></script>
   <script src="https://unpkg.com/cartogram-chart"></script>
</head>
<body>
   <div class="form-group">
      <label class="control-label" for="select">Value:</label>
      <div>
         <select id="select" class="form-control">
            <option value="POP_EST" selected>POP_EST</option>
            <option value="GDP_MD_EST">GDP_MD_EST</option>
         </select>
      </div>
   </div>
   <div id="world"></div>
   <script>
      var carto;
      
            d3.json('./ne_110m_admin_0_countries.json', (error, world) => {
                if (error) throw error;
      
                // exclude antarctica
                world.objects.countries.geometries.splice(
                    world.objects.countries.geometries.findIndex(d => d.properties.ISO_A2 === 'AQ'),
                    1
                );
      
                const colorScale = d3.scaleOrdinal([...d3.schemeCategory20, ...d3.schemeCategory20b, ...d3.schemeCategory20c]);
      
                carto = Cartogram()
                    .topoJson(world)
                    .topoObjectName('countries')
                    .iterations(60)
                    .value(({ properties }) => properties.POP_EST)
                    .color(({ properties: { ISO_A2 } }) => colorScale(ISO_A2))
                    .label(({ properties: p }) => `Population of ${p.NAME} (${p.ISO_A2})`)
                    .valFormatter(d3.format(".3s"))
                    .onClick(d => console.info(d))
                    (document.getElementById('world'));
            });
      
      document.getElementById("select").addEventListener("change", function() {
      	var selectValue = document.getElementById("select").value;
      	carto
      	   .value(function(d) { return d.properties[selectValue]; })
      	   .label(function(d) {
      		  return "Population of" + d.properties.name + "(" + d.properties[selectValue] + ")";
      		})
      }, false);
        
   </script>
</body>
```

**Comment by vasturiano** [comment]
@pvictor you're right, there was a bug in the lib, in which changes to `label` or `tooltipContent` were not being taken into account. This is now fixed, so your example above should work in `v1.0.6`.
Thanks for the report!

**Comment by pvictor** [comment]
Amazing ! Many thanks !
